### DISCUSSION TITLE
Make all fields localizable by default

### DISCUSSION THREAD

**Discussion by carstenjaksch** [studio1902/statamic-peak-seo#45] [Ideas]
For example, currently it is not possible to set different analytics providers for different websites by default.

It is an annoying extra step to manually set all fields to localizable after installing the starter kit.

**Comment by robdekort** [comment]
I've only ever made the fields localizable by default that to me make sense tot localize, but if this is something you need you could PR it in.

**Reply by carstenjaksch** [reply]
Alright, the PR editor told me to discuss this first.

**Reply by robdekort** [reply]
Yep, thanks. Happy to merge it in.
